### PR TITLE
[Doc] PR for Document gflag validation support (Doc-PLAT-2474)

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-2474

Preview: https://deploy-preview-11155--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-platform/manage-deployments/edit-config-flags/

### CODE REVIEW

**Comment by lizayugabyte** [comment]
@Lingeshwar Could you review the doc, please?

**Review by Lingeshwar — CHANGES_REQUESTED** [review]
@lizayugabyte , Here is my feedback

1. How to add gflags in create Universe ? (Missing here).
2. I have updated modal styles used here (Better to use that in documentation) because it has more real estate and clean.
3. Add Flags through free text is missing ? (For Bulk flag import).

rest LGTM

**Comment by lizayugabyte** [comment]
@Lingeshwar I can't open the Add to Master dialog from Create Universe on dev server because it never loads:
Screen Shot 2022-01-25 at 3 14 47 PM

**Comment by lizayugabyte** [comment]
@Lingeshwar 
The "Add flags via Text" description was included in the first version of the doc. 
Added a section on adding flags to a new universe.
Can't change the screen shots because the UI doesn't load.

**Comment by lizayugabyte** [comment]
Via slack today:
Lingesh  9:34 AM
New changes are not yet deployed in portal
Add as Free text option is now going to replace with Add as JSON and ui is not similar with earlier ui.

**Comment by Lingeshwar** [comment]
@lizayugabyte , Sorry for the inconvenience caused. 
Today I have been asked to make the changes from "ADD as free text" to "Add as JSON". 
I will let you know once it is done and I will send you all the new screenshots.
Thank you

**Review by Lingeshwar — COMMENTED** [review]
@lizayugabyte , Can you please provide some information about Add as JSON option and attach it's screenshot.
rest LGTM

Check the below image
Screenshot 2022-02-04 at 8 48 07 AM

File: docs/content/latest/yugabyte-platform/manage-deployments/edit-config-flags.md [hunk 1]
```diff
@@ -15,22 +15,48 @@ isTocNested: true
...
+
+Use the **Flags** dialog to do all or some of the following, as per your requirements:
+
+- Click **Add Flags > Add to Master** to open the **Add to Master** dialog, then select the flag you want to add to YB-Master and set its value, as per the following illustration:<br><br>
+  
+  ![Master Config](/images/ee/add-master-1.png)<br><br>
+
+- Click **Add Flags > Add to T-Server** to open the **Add to T-Server** dialog, then select the flag you want to add to YB-TServer and set its value.
+
+- Use **Add as JSON > Add to Master** and **Add as JSON > Add to T-Server** to import flags in bulk. The flugs must be defined as name-value pairs in a JSON format via the in the **Add to T-Server** or **Add to Master** dialog, as per the following illustration:<br><br>
```
Comment (Lingeshwar) [inline docs/content/latest/yugabyte-platform/manage-deployments/edit-config-flags.md:38]: Just typo change
change flugs to flags and name-value pairs to key-value pairs

**Review by Lingeshwar — CHANGES_REQUESTED** [review]

**Review by Lingeshwar — APPROVED** [review]
LG

**Pull request merged**